app/docs/dev/policies/meshaccesslog.md

```diff
@@ -0,0 +1,287 @@
+
+The shape of a single log record is defined by a template string that uses [command operators](https://www.envoyproxy.io/docs/envoy/v1.22.0/configuration/observability/access_log/usage#command-operators) to extract and format data about a `TCP` connection or an `HTTP` request.
+
+E.g.,
+
+```
+%START_TIME% %KUMA_SOURCE_SERVICE% => %KUMA_DESTINATION_SERVICE% %DURATION%
+```
+
+where `%START_TIME%` and `%KUMA_SOURCE_SERVICE%` are examples of available _command operators_.
+
+All _command operators_ [defined by Envoy](https://www.envoyproxy.io/docs/envoy/v1.22.0/configuration/observability/access_log/usage#command-operators) are supported, along with additional _command operators_ defined by Kuma:
+
+| Command Operator                     | Description                                                      |
+|--------------------------------------|------------------------------------------------------------------|
+| `%KUMA_MESH%`                        | name of the mesh in which traffic is flowing                     |
+| `%KUMA_SOURCE_SERVICE%`              | name of a `service` that is the `source` of traffic              |
+| `%KUMA_DESTINATION_SERVICE%`         | name of a `service` that is the `destination` of traffic         |
+| `%KUMA_SOURCE_ADDRESS_WITHOUT_PORT%` | address of a `Dataplane` that is the `source` of traffic         |
+| `%KUMA_TRAFFIC_DIRECTION%`           | direction of the traffic, `INBOUND`, `OUTBOUND` or `UNSPECIFIED` |
+
+All additional access log _command operators_ are valid to use with both `TCP` and `HTTP` traffic.
+
+If a _command operator_ is specific to `HTTP` traffic, such as `%REQ(X?Y):Z%` or `%RESP(X?Y):Z%`, it will be replaced by a symbol "`-`" in case of `TCP` traffic.
+
+Internally, Kuma [determines traffic protocol](/docs/{{ page.version }}/policies/protocol-support-in-kuma) based on the value of `kuma.io/protocol` tag on the `inbound` interface of a `destination` `Dataplane`.
+
+There are two types of `format` - `plain` and `json`.
+
+Plain will accept a string with _command operators_ and produce a string output.
+
+JSON will accept a list of key-value pairs that will produce a valid JSON object.
+
+#### Plain
+
+The default format string for `TCP` traffic is:
+
+```
+[%START_TIME%] %RESPONSE_FLAGS% %KUMA_MESH% %KUMA_SOURCE_ADDRESS_WITHOUT_PORT%(%KUMA_SOURCE_SERVICE%)->%UPSTREAM_HOST%(%KUMA_DESTINATION_SERVICE%) took %DURATION%ms, sent %BYTES_SENT% bytes, received: %BYTES_RECEIVED% bytes
+```
+
+The default format string for `HTTP` traffic is:
+
+```
+[%START_TIME%] %KUMA_MESH% "%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%" %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% "%REQ(X-FORWARDED-FOR)%" "%REQ(USER-AGENT)%" "%REQ(X-REQUEST-ID)%" "%REQ(:AUTHORITY)%" "%KUMA_SOURCE_SERVICE%" "%KUMA_DESTINATION_SERVICE%" "%KUMA_SOURCE_ADDRESS_WITHOUT_PORT%" "%UPSTREAM_HOST%"
+```
```

Comment:
So how do we pick from these 2 types?

Reply:
A few paragraphs above we describe that we use `kuma.io/protocol` - https://github.com/kumahq/kuma-website/pull/1097/files#diff-fee054c5265ec14fa4800c086675e7fedc48eb57d8dc2a6e37b5f0005e61e485R58

Or do you mean between plain vs json?

Reply:
I mean plain vs json

Reply:
I'm guessing this is user preference or the backend might actually require `JSON`. I'll add a sentence here.